docs/release-notes/next-minerva/PT-162792344-new-aevm-version.md

```diff
@@ -0,0 +1,2 @@
+* Introduces a new AEVM version to contain consensus breaking changes and optimizations.
```

Comment:
I expect this follows https://github.com/aeternity/epoch/pull/1964#pullrequestreview-187131056
Thanks.